Extraction/Classes/ARLoadFailureView.m

```diff
@@ -105,21 +106,15 @@ - (void)rotate;
                             }
                             completion:^(BOOL _) {
                               __strong __typeof(weakSelf) strongSelf = weakSelf;
-                              if (strongSelf) {
-                                if (strongSelf.stopRotating) {
-                                  strongSelf.stopRotating = NO;
-                                } else {
-                                  [strongSelf rotate];
-                                }
```

Comment:
3) `[strongSelf rotate]` would not get called again because `stopRotating` was `YES` and thus the animation would stop after 1 iteration.